src/ui/windows/TogglDesktop/TogglDesktop/SingleInstanceManager.cs

```diff
@@ -12,6 +12,7 @@ public SingleInstanceManager()
         }
 
         public event Action BeforeStartup;
+    public event Action BeforeStartup;

```

Comment:
Why is there two of the same thing here?

Reply:
I'm not sure what the cs file is edited. I will revert the change.